lib/api/BackbeatAPI.js

```diff
@@ -51,6 +51,7 @@ class BackbeatAPI {
         this._queuePopulator = config.queuePopulator;
         this._kafkaHost = config.kafka.hosts;
         this._redisConfig = config.redis;
+        this._localCacheRedis = config.localCache;
```

Comment:
Naming consistency suggestion: I would rather call `_localRedisConfig` here (or `_localCacheConfig`), and the `RedisClient` object, `this._localRedis` (or `_localCache` / `_localCacheClient`). Let's keep for later.